src/styles.scss

```diff
@@ -787,6 +787,10 @@ div.side-block img {
   vertical-align: middle;
 }
 
+.mat-icon-copy-button {
```

Comment:
should be something generic like small-mat-icon or mat-icon-small or even more generic for potential re-use